Main.py

```diff
@@ -1,26 +1,71 @@
-from Register import Register
+from QuantumRegister import *
 from Qubit import Qubit
 import Operators as op
-import Tests
 
-# this main runs through a few of the materials covered in the notes
-# the three seven tests is covered in the slides
-print("REGISTER STATE: VECTOR AS VERTICAL MATRIX")
-r1 = Tests.threeSevenTest()
-Tests.normalizedTest(r1)
-r1.updateS()
+import deutsch
+import deutschJozsa

```

Comment:
from deutsch import * 
allows us to access the class without calling file name 
same for others